src/libcore/cmp.rs

```diff
@@ -32,6 +31,66 @@
 //! assert_eq!(x == y, false);
 //! assert_eq!(x.eq(&y), false);
 //! ```
+//!
+//! This module also provides the following two traits:
+//!
+//! * [`Eq`]: which requires `PartialEq`, states that all values of a type can be
+//!   compared to all others,
+//! * [`Ord`]: which requires `PartialOrd` and `Eq`, states that the ordering
+//!   relations expressed by the comparison operators form all total ordering relations 
+//!   (more on this below).
+//!
+//! The distinction between these traits is important: it tell us what it _means_ for a 
+//! sequence of values to be ordered.
```

Comment:
This is the pragmatic way I choose to put it, might not be the best one. 

@LukasKalbertodt I basically decided to put the "intro docs" to `PartialOrd` at the top level of the module. Otherwise the docs of `PatialOrd` and other traits end up restating the same things over and over again and is hard to keep them in sync.

Reply:
The second part of this sentence does not talk about a distinction, so I am confused what this is even telling us.

Reply:
Actually I am not even sure which distinction you mean: `PartialEq` vs `Eq`, or `Eq` vs `Ord`...